client/my-sites/stats/stats-module/all-time-nav.jsx

```diff
@@ -69,66 +72,82 @@ export const StatsModuleSummaryLinks = ( props ) => {
 		recordStats( item );
 	};
 
-	// Template for standard range options (7, 30, Quarter, Year, All Time).
-	const summaryPath = `/stats/day/${ path }/${ siteSlug }?startDate=${ moment().format(
-		'YYYY-MM-DD'
-	) }&summarize=1&num=`;
+	const getSummaryPath = useMemo( () => {
+		return ( numberDays ) => {
+			let url = window.location.href;
+			let queryParams = new URLSearchParams( url.split( '?' )[ 1 ] || '' );
 
-	// Path for summary or custom range option. ie: The first button in the row.
-	// Defaults to one day/week/month/year.
-	let summaryPeriodPath = `/stats/${
-		period.period
-	}/${ path }/${ siteSlug }?startDate=${ period.endOf.format( 'YYYY-MM-DD' ) }`;
-	// Override if custom range was used in query.
-	if ( query.start_date ) {
-		summaryPeriodPath = `/stats/${ period.period }/${ path }/${ siteSlug }?startDate=${ query.start_date }&endDate=${ query.date }`;
-	}
+			if ( isOdysseyStats ) {
+				url = window.location.href;
+				const hash = window.location.hash.split( '#!' )[ 1 ];
+				const hashQuery = new URLSearchParams( hash.split( '?' )[ 1 ] || '' );
+				queryParams = new URLSearchParams( hashQuery );
+			}
+
+			queryParams.set( 'startDate', moment().format( 'YYYY-MM-DD' ) );
+			queryParams.set( 'summarize', 1 );
+			queryParams.set( 'num', numberDays );
+			queryParams.delete( 'endDate' );
+
+			if ( numberDays === 'day' ) {
+				if ( query.start_date ) {
+					queryParams.set( 'startDate', query.start_date );
+					queryParams.set( 'endDate', query.date );
+				}
+
+				queryParams.delete( 'num' );
```

Comment:
Correct me if I'm missing anything - I thought summary page should always have the summarize parameter?

Reply:
The UTM API endpoint does not actually need to have the `summarized` parameter: fbhepr%2Skers%2Sjcpbz%2Schoyvp.ncv%2Serfg%2Sjcpbz%2Qwfba%2Qraqcbvagf%2Spynff.jcpbz%2Qfgngf%2Qncv%2Qraqcbvagf.cuc%3Se%3Q991p62rs%231677-og.